libs/json/src/test/test_json_simple.c

```diff
@@ -180,6 +181,23 @@ TEST_CASE(test_json_simple_decode){
     long long int intarr[8];
     int rc;
     int array_count;
+    
+    /*working with structures of arrrays. declearing variable*/
+    
+    struct test_structarr {
+        
```

Comment:
Nges: first, thank you for the work!  
Second: please don't put a sapce between the definition of the structure test_structarr, and the field defintions.